### ISSUE TITLE
UselessParentheses false positive

### ISSUE THREAD

**Issue by simPod** [slevomat/coding-standard#797]
UselessParentheses sniff requires this change:

```diff
- $position + 1 - ($match[1] + mb_strlen($match[0], 'UTF-8'));
+ $position + 1 - $match[1] + mb_strlen($match[0], 'UTF-8');
```

That changes logic in the code though.

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/263d3a9d85604c0ad9cd1c4283dd3f279127027e

**Comment by simPod** [comment]
❤️ quick fix, thanks!
